static/js/components/PrivacyForm.js

```diff
@@ -0,0 +1,26 @@
+/* global SETTINGS: false */
```

Comment:
could you add flow typechecking to this file? you can do so by adding a comment on the first line like this:

```js
// @flow
```